sormas-ui/src/main/java/de/symeda/sormas/ui/reports/aggregate/AggregateReportsFilterForm.java

```diff
@@ -124,58 +128,61 @@ protected void addFields() {
 
 	@Override
 	protected void applyDependenciesOnFieldChange(String propertyId, Property.ValueChangeEvent event) {
-		super.applyDependenciesOnFieldChange(propertyId, event);
 
-		final ComboBox districtField = getField(AggregateReportCriteria.DISTRICT);
-		final ComboBox facilityField = getField(AggregateReportCriteria.HEALTH_FACILITY);
-		final ComboBox pointOfEntryField = getField(AggregateReportCriteria.POINT_OF_ENTRY);
+		super.applyDependenciesOnFieldChange(propertyId, event);
 
 		final UserDto user = currentUserDto();
 
 		switch (propertyId) {
 		case AggregateReportCriteria.REGION: {
 			final RegionReferenceDto region = user.getRegion() != null ? user.getRegion() : (RegionReferenceDto) event.getProperty().getValue();
 			if (region != null) {
-				enableFields(districtField);
-				FieldHelper.updateItems(districtField, FacadeProvider.getDistrictFacade().getAllActiveByRegion(region.getUuid()));
-				facilityField.clear();
-				pointOfEntryField.clear();
+				boolean districtFieldEnabled = districtFilter.isEnabled();
```

Comment:
this boolean variable should be removed. It influences the behavior in the case of National User. In this particular case, the District becomes readOnly no matter what is selected on the Region ComboBox.